package.json

```
@@ -15,17 +15,17 @@
     "react-confetti": "^6.1.0",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.3.0",
-    "react-scripts": "5.0.0",
     "web-vitals": "^2.1.2"
   },
+  "type": "module",
   "scripts": {
-    "predeploy": "npm run build && cp CNAME build/CNAME",
+    "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
```

since we are using the default dist folder, this should be updated too

```suggestion
    "deploy": "gh-pages -d dist",
```

---

is there something i forgot?

---

No, it looks like all of the comments have been resolved.
The whole freeCodeCamp team is on Christmas break.
So I will take look at this PR after break 👍